modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/KnowledgeBaseArticleResourceTest.java

```diff
@@ -113,6 +164,16 @@ protected KnowledgeBaseArticle randomKnowledgeBaseArticle()
 		KnowledgeBaseArticle knowledgeBaseArticle =
 			super.randomKnowledgeBaseArticle();
 
+		Date date1 = RandomTestUtil.nextDate();
+
+		Instant instant1 = date1.toInstant();
+
+		Instant instant2 = instant1.truncatedTo(ChronoUnit.SECONDS);
+
+		Date date2 = Date.from(instant2);
```

Comment:
When a variable is used only once (and there's no chaining), it has to be inlined.
```
knowledgeBaseArticle.setDatePublished(Date.from(instant1.truncatedTo(...)));
```
After this, you should remove the `1` suffix from the variables.